<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Zaporra - Administración de Socios</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Búsqueda</string>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLineEdit" name="filterPatternLineEdit" />
         </item>
         <item>
          <widget class="QRadioButton" name="nameAndSurnameRB" >
           <property name="text" >
            <string>Nombre y Apellidos</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="noRB" >
           <property name="text" >
            <string>Número de Socio</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="dniRB" >
           <property name="text" >
            <string>DNI</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="memberDetailsBtn" >
        <property name="text" >
         <string>Ver &amp;detalles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newUserBtn" >
        <property name="text" >
         <string>&amp;Nuevo Socio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitBtn" >
        <property name="text" >
         <string>&amp;Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="memberView" >
      <property name="minimumSize" >
       <size>
        <width>666</width>
        <height>240</height>
       </size>
      </property>
      <property name="horizontalScrollBarPolicy" >
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers" >
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo" >
    <property name="title" >
     <string>&amp;Archivo</string>
    </property>
   </widget>
   <addaction name="menuArchivo" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
